### PR TITLE
use git lfs for mp4 and webm

### PR DESCRIPTION
**Pull request by felicio**

### CODE REVIEW

File: .gitattributes
Comment (felicio) [inline .gitattributes:1]: usage:

## Vercel

image

https://vercel.com/status-im-web/status-website/settings/git

## localhost

image
https://docs.github.com/en/repositories/working-with-files/managing-large-files/configuring-git-large-file-storage

File: public/assets/wallet/vitalik.mp4
Comment (felicio) [inline public/assets/wallet/vitalik.mp4:1]: follow-up: https://docs.github.com/en/repositories/working-with-files/managing-large-files/moving-a-file-in-your-repository-to-git-large-file-storage

File: .gitattributes
Comment (felicio) [reply .gitattributes:1]: ## example

```
felicio@Felicios-MacBook-Pro status-website % git push -u origin f/lfs
Uploading LFS objects: 100% (8/8), 252 MB | 9.5 MB/s, done.    
```

**Review by prichodko — APPROVED** [review]
any chance to keep the `vitalik` file name? Otherwise LFG.

**Review by marcelines — APPROVED** [review]

**Comment by felicio** [comment]
> any chance to keep the `vitalik` file name? Otherwise LFG.

not in this pr, had to remove the committed one first

File: .gitattributes
Comment (felicio) [reply .gitattributes:1]: ## install

image

https://docs.github.com/en/repositories/working-with-files/managing-large-files/installing-git-large-file-storage

**Pull request merged**